controllers/osbuildenvconfig_controller.go

```diff
@@ -87,7 +87,8 @@ const (
 	workerAPIInternalPort   = 18700
 	workerAPIExternalPort   = 8700
 
-	envoyProxyCertsDir = "/etc/certs"
+	envoyProxyCertsDir   = "/etc/certs"
+	EnvoyOSBuildCertsDir = "/etc/osbuild/certs"
```

Comment:
First, this const does not belong here as it is not used by this controller. Second, the name is incorrect since it is used to find the Operator's certificates inside the Operator's container. So, its names should not have `Envoy` in it (same as the strings for the worker certificate do not mention envoy)